web_responsive/static/tests/js/web_responsive.js

```diff
@@ -1,278 +1,318 @@
 /* Copyright 2016 LasLabs Inc.
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 
-odoo.define_section('web_responsive', ['web_responsive'], function(test) {
+odoo.define('web_responsive.test', function(require) {
     "use strict";
-    
-    // It provides a base drawer compatible interface for testing
-    self.initInterface = function(AppDrawer) {
-        
-        var $el = $('<div class="drawer drawer--left">');
-        $el.append(
-            $('<header role="banner">')
-                .append(
-                    $('<button class="drawer-toggle"><span class="drawer-hamburger-icon">')
-                )
-                .append(
-                    $('<nav class="drawer-nav"><ul class="drawer-menu"><li class="drawer-menu-item">')
-                )
+
+    var core = require('web.core');
+    var responsive = require('web_responsive');
+
+    QUnit.module('web_responsive', {
+        beforeEach: function() {
+            var $el = $('<div class="drawer drawer--left">')
```

Comment:
Still all of these appends seem overly complex. Can't you write normal HTML and put it into this first call?

Reply:
I'm not saying it won't work, but I really hope I wouldn't have done something this incredibly complex and stupid for no reason.

Reply:
So yeah guess I did do it for no reason. Thanks @hgminh95!